app/components/polaris/popover_component.html.erb

```diff
@@ -7,25 +7,27 @@
     <% end %>
   </div>
 
-  <%= render Polaris::BaseComponent.new(**system_arguments) do %>
-    <%= render Polaris::BaseComponent.new(**popover_arguments) do %>
-      <div class="Polaris-Popover__FocusTracker" tabindex="0"></div>
-      <div style="">
-        <div class="Polaris-Popover__Wrapper">
-          <%= render Polaris::BaseComponent.new(**content_arguments) do %>
-            <% if panes.present? %>
-              <% panes.each do |pane| %>
-                <%= pane %>
-              <% end %>
-            <% else %>
-              <%= render Polaris::Popover::PaneComponent.new(sectioned: @sectioned) do %>
-                <%= content %>
+  <div data-polaris-popover-target="template" style="display: none;">
```

Comment:
Should we implement system tests first so that we can replace this `<div>` with a `<template>`?

Reply:
Let's add system tests first. I'm already working on it.

Reply:
Awesome 👍

Reply:
@yourivdlans system tests are ready: https://github.com/baoagency/polaris_view_components/pull/224
Let's update this to `<template>`? 🙂

Reply:
Yes! Will do 😄